polygon/sync/block_downloader.go

```diff
@@ -82,6 +87,7 @@ type blockDownloader struct {
 	logger                        log.Logger
 	p2pService                    p2p.Service
 	heimdall                      heimdall.Heimdall
+	bridge                        *bridge.Bridge
```

Comment:
It is preferable to use interfaces for external component dependencies because interfaces can be mocked for unit/integration tests.
We're using interfaces here for everything else, so it would be nice if we can stick to that and replace this with an interface.